pkgs/applications/networking/irc/qweechat/default.nix

```diff
@@ -1,11 +1,11 @@
 { stdenv, fetchFromGitHub, python27Packages }:
 
 python27Packages.buildPythonApplication rec {
+  pname = "qweechat-unstable";
   version = "2016-07-29";
-  name = "qweechat-unstable-${version}";
   namePrefix = "";
```

Comment:
Is this related?

Reply:
Related to what?

Reply:
No, that just strips `python3.7-` prefix added by `buildPythonPackage`.

Reply:
:+1:.